.github/workflows/docker.yml

```
@@ -0,0 +1,43 @@
+name: Docker Build and Push
+
+on:
+  push:
+    branches:
+      - main
```

Kita pakai master di sini kak.

```suggestion
      - master
```